include/finufft/finufft_core.h

```diff
@@ -95,6 +95,17 @@ inline constexpr BIGINT MAX_NF = BIGINT(1e12);
 // values for M = nj (also nk in type 3)...
 inline constexpr BIGINT MAX_NU_PTS = BIGINT(1e14);
 
+// MR: In the longer term I suggest to move
+// away from M_PI, which was never part of the standard.
+// Perhaps a constexpr pi in the namespace finufft, or a constexpr finufft_pi
```

Comment:
We could do that even now.

Reply:
Happy to do so! We just need to find a name everyone is happy with, and we cannot overrie `M_PI`, since that is a macro (if the systen provides it).
Using just a `double` constant `PI` should be fine, as long as we only have it in the internal header files.

Reply:
Yes, and we cast it to TF where is used. Unless we want to make a constexpr PI function.

Reply:
Sorry, it's already done in this PR now ...